tools/Makefile.shared

```diff
@@ -25,43 +83,38 @@ else
 export OCAML_FLEXLINK:=$(ROOTDIR)/boot/ocamlrun $(ROOTDIR)/flexdll/flexlink.exe
 endif
 
-CAMLC=$(CAMLRUN) ../boot/ocamlc -nostdlib -I ../boot -use-prims ../byterun/primitives
-CAMLOPT=$(CAMLRUN) ../ocamlopt -nostdlib -I ../stdlib
+CAMLC=$(CAMLRUN) ../boot/ocamlc -nostdlib -I ../boot -use-prims \
+  ../byterun/primitives -I ..
```

Comment:
Why do we need to add `-I ..` ? AFAICT, there shouldn't be any `.cmi` files in `..`